routers/web/feed/convert.go

```diff
@@ -52,7 +52,7 @@ func toReleaseLink(act *activities_model.Action) string {
 func renderMarkdown(ctx *context.Context, act *activities_model.Action, content string) string {
 	markdownCtx := &markup.RenderContext{
 		Ctx:       ctx,
-		URLPrefix: act.GetRepoLink(),
+		URLPrefix: setting.AppSubURL,
```

Comment:
I don't know how this is used and if the change is correct.

Reply:
IIRC, it's used for the relative URLs in markdown. For example, if you put a relative URL in markdown, it will be based on the URLPrefix (aka the current repo).

So, I think the old behavior is expected.